### ISSUE TITLE
Minecraft server worlds wont unzip

### ISSUE THREAD

**Issue by BANanaD3V** [IogaMaster/dynamo#6]
**Describe the bug**
When using mcman worlds feature with a world zip, the server won't build and will throw `permission denied (os error 13)` when on world stage.

**To Reproduce**
1. Create a mcman server
2. Add a world zip and `[worlds.worldName]` to server.toml
3. Try to build the server with flux
4. Error occurs

**Expected behavior**
A world would unzip with no errors

**Screenshots**
![image](https://github.com/user-attachments/assets/c4c53447-fcd0-4968-82dd-034f39587dec)

**Additional context**
This might be due to how mcman unzips worlds, but unsure.

**Comment by IogaMaster** [comment]
I will investigate

**Comment by BANanaD3V** [comment]
Any updates?

**Comment by ProggerX** [comment]
@IogaMaster commit that "fixed this issue" breaks all flux and i have "can't mkdir server_build: permission denied"
I think buildPhase is not allowed to edit $src dir contents

**Comment by IogaMaster** [comment]
> @IogaMaster commit that "fixed this issue" breaks all flux and i have "can't mkdir server_build: permission denied"
> 
> I think buildPhase is not allowed to edit $src dir contents

Reverted the commit

**Comment by hustlerone** [comment]
Is specifying custom world dirs supported?

**Comment by IogaMaster** [comment]
Anything that mcman can do, flux can do.
I am unaware of such feature, but check out the mcman source to be sure

**Comment by hustlerone** [comment]
so I can use any folder as the working directory?

**Comment by IogaMaster** [comment]
yeah, flux just takes the path to the mcman config in a subdir. The mcman binary then generates a server from that. So do as you please in the mcman config
